rotkehlchen/types.py

```diff
@@ -347,7 +347,7 @@ class SupportedBlockchain(SerializableEnumValueMixin):
     OPTIMISM = 'OPTIMISM'
     BINANCE = 'BINANCE'
     GNOSIS = 'GNOSIS'
-    MATIC = 'MATIC'
+    POLYGON = 'MATIC'
```

Comment:
keep it as it was. Matic is the name of the chain and polygon is the general project. Also as the value is still `MATIC` makes no sense to rename